src/api/projectSteps.ts

```diff
@@ -0,0 +1,62 @@
+enum ProjectStatus {
+  InDevelopment = 'InDevelopment',
+  Pending = 'Pending',
+  Active = 'Active',
+  Stopped = 'Stopped',
+  Finished = 'Finished',
+}
+
+enum ProjectStep {
```

Comment:
Both of these enums can be replaced with the string literals from generated schema.

Reply:
Changed